config.json

```diff
@@ -18,5 +18,9 @@
   },
   "proxy": "",
   "AWS_ACCESS_KEY_ID": "XXXXXXXXXXXXXXXXX",
-  "AWS_SECRET_ACCESS_KEY": "XXXXXXXXXXXXXXXXXXXXX"
+  "AWS_SECRET_ACCESS_KEY": "XXXXXXXXXXXXXXXXXXXXX",
```

Comment:
@aayusharora Yes we should have it on readme.md, will help new comers a lot.
@Princu7 What do you say?